bundle/deploy/files/sync.go

```diff
@@ -19,17 +27,22 @@ func getSync(ctx context.Context, b *bundle.Bundle) (*sync.Sync, error) {
 		return nil, fmt.Errorf("cannot get list of sync includes: %w", err)
 	}
 
-	opts := sync.SyncOptions{
+	opts := &sync.SyncOptions{
 		LocalPath:  b.Config.Path,
 		RemotePath: b.Config.Workspace.FilePath,
 		Include:    includes,
 		Exclude:    b.Config.Sync.Exclude,
+		Host:       b.WorkspaceClient().Config.Host,
 
-		Full:        false,
-		CurrentUser: b.Config.Workspace.CurrentUser.User,
+		Full: false,
```

Comment:
Good point on full sync, updated. We can't really keep the local snapshot as @pietern pointed earlier for cases when someone else have made a deployment with new files. If we keep local snapshot we will just lose track of these files.

Reply:
It will be a full sync always because the mtimes are zero-initialized. Every file will therefore be newer.

Note that this is the case IFF the remote deployment state was newer than the local one; i.e. someone did a deploy after we did our last deploy locally. That immediately invalidates our local sync snapshot.